cluster/addons/cluster-monitoring/influxdb/heapster-controller.yaml

```diff
@@ -20,7 +20,7 @@ spec:
         kubernetes.io/cluster-service: "true"
     spec:
       containers:
-        - image: gcr.io/google_containers/heapster:v0.18.1
+        - image: thuctnguyen/heapster:v0.18.2-rc1
```

Comment:
Not yet. We can make an official heapster release with InfluxDB v0.9.x once
your PR lands.

On Wed, Sep 23, 2015 at 11:22 AM, Thuc Nguyen <notifications@github.com>
wrote:

> In cluster/addons/cluster-monitoring/influxdb/heapster-controller.yaml
> <https://github.com/kubernetes/kubernetes/pull/14423#discussion_r40238516>
> :
>
> > @@ -20,7 +20,7 @@ spec:
> >          kubernetes.io/cluster-service: "true"
> >      spec:
> >        containers:
> > -        - image: gcr.io/google_containers/heapster:v0.18.1
> > +        - image: thuctnguyen/heapster:v0.18.2-rc1
>
> This PR depends on the latest heapster code with support for InfluxDB
> 0.9.4. I built heapster from HEAD and created
> thuctnguyen/heapster:v0.18.2-rc1 from that binary. Is there a Docker image
> for the latest heapster code that I can use?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/14423/files#r40238516>.
>